src/templates/fr/2020/methodology.html

```diff
@@ -0,0 +1,426 @@
+<span class="code-keyword">FROM</span>
+  <span class="code-string">`httparchive.summary_pages.2019_07_01_*`</span>,
+  <span class="code-function">UNNEST</span>([<span class="code-number">10</span>, <span class="code-number">25</span>, <span class="code-number">50</span>, <span class="code-number">75</span>, <span class="code-number">90</span>]) <span class="code-keyword">AS</span> percentile
+<span class="code-keyword">GROUP BY</span>
+  percentile,
+  client
+<span class="code-keyword">ORDER BY</span>
+  percentile,
+  client</code></pre>
+        </div>
+
+        <p>
+        Ces résultats statistiques peuvent être consultés publiquement dans des feuilles de calcul propres à chaque chapitre, par exemple voici <a href="https://docs.google.com/spreadsheets/d/1kBTglETN_V9UjKqK_EFmFjRexJnQOmLLr-I2Tkotvic/edit?usp=sharing">les résultats pour JavaScript</a>.
+        </p>
+    </section>
+
+    <section>
+        <h3 id="websites"><a href="#websites" class="anchor-link">Sites Web</a></h3>
+
+        <p>
+        Il y a 7&nbsp;546&nbsp;709 sites Web dans l’ensemble de données. Parmi ceux-ci, 6&nbsp;347&nbsp;919 sont des sites Web mobiles et 5&nbsp;593&nbsp;642 sont des sites Web destinés aux ordinateurs de bureau. La plupart des sites Web sont inclus dans les deux sous-ensembles à la fois.
+        </p>
+
+        <p>
+        HTTP Archive puise les URLs à tester dans le <a href="#chrome-ux-report">Chrome UX Report</a>. Chrome UX Report est un jeu de données public appartenant à Google qui regroupe des informations sur l’expérience des utilisateurs et utilisatrices de Chrome lors de leurs visites sur des millions de sites Web. Il nous fournit une liste de sites Web qui sont à jour et qui reflètent l’utilisation du Web dans le monde réel. Le jeu de données du Chrome UX Report comprend une dimension relative au type de matériel utilisé, que nous exploitons pour savoir si les sites sont consultés sur ordinateurs de bureau ou périphériques mobiles.
+        </p>
+
+        <p>
+        Le parcours de ces sites, réalisé par HTTP Archive en août 2020 et utilisé par le Web Almanac, a utilisé la dernière version disponible du Chrome UX Report, juin 2020 (201905), pour sa liste de sites Web. Cet ensemble de données a été publié le 14 juillet 2020 et reflète les sites Web visités par les utilisateurs de Chrome pendant le mois de mai.
+        </p>
+
+        <!-- To be translated <p>
+          There was was a 20-30% growth on websites analysed compared to the <a href="../2019/methodology#websites">number of sites analysed for the 2019 Web Almanac</a>. This increase has been analysed by <a href="{{ url_for('contributors', year=year, lang=lang, _anchor='paulcalvano') }}">Paul Calvano</a> in his <a href="https://paulcalvano.com/2020-09-29-growth-of-the-web-in-2020/">Growth of the Web in 2020</a> post.
+        </p> -->
```

Comment:
Can a native speaker translate this and remove the `<!--` and `-->` marks?

Reply:
```suggestion
        <p>
                Il y a eu une croissance de 20 à 30 % des sites web analysés par rapport au <a href="../2019/methodology#websites">nombre de sites analysés pour le Web Almanac 2019</a>. Cette augmentation a été analysée par <a href="{{ url_for('contributors', year=year, lang=lang, _anchor='paulcalvano') }}}">Paul Calvano</a> dans son article <a href="https://paulcalvano.com/2020-09-29-growth-of-the-web-in-2020/" lang="en">Growth of the Web in 2020</a>.
        </p>
```